reconcile/ocm_additional_router.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import logging
+import json
+
+import reconcile.queries as queries
+
+from reconcile.status import ExitCodes
+from reconcile.utils.ocm import OCMMap
+
+QONTRACT_INTEGRATION = 'ocm-additional-router'
+
+
+def fetch_current_state(clusters):
+    settings = queries.get_app_interface_settings()
+    ocm_map = OCMMap(clusters=clusters, integration=QONTRACT_INTEGRATION,
+                     settings=settings)
+
+    current_state = []
+    for cluster in clusters:
+        cluster_name = cluster['name']
+        ocm = ocm_map.get(cluster_name)
+        routers = ocm.get_additional_routers(cluster_name)
+        for router in routers:
+            router['cluster'] = cluster_name
+            current_state.append(router)
+
+    return ocm_map, current_state
+
+
+def fetch_desired_state(clusters):
+    desired_state = []
+    for cluster in clusters:
+        cluster_name = cluster['name']
+        router = cluster['additionalRouter']
+        listening = 'internal' if router['private'] else 'external'
+        item = {
+            'listening': listening,
+            'cluster': cluster_name
+        }
+        selectors = cluster['additionalRouter']['route_selectors']
```

Comment:
This is `router['route_selectors']`, isn't it? Either use `router` all the way or drop this variable and use `cluster['additionalRouter']`. In any case, do the same things the same way always.